### ISSUE TITLE
every subject should have a user set as admin

### ISSUE THREAD

**Issue by brunoamaral** [brunoamaral/gregory#224]
- [x] admins should have the option to create new subjects
- [ ] admins should received admin-digests for the subjects they selected
- [ ] ~email lists should be linked to one or more subjects~

**Comment by brunoamaral** [comment]
Putting this on hold because there isn't a need at the moment.

**Comment by brunoamaral** [comment]
depends on #329

**Comment by brunoamaral** [comment]
no longer needed because we now use Teams to handle notifications. Granular permissions of this kind are not a priority right now.
